src/applications/static-pages/shared/sass/_m-operating-status.scss

```diff
@@ -0,0 +1,77 @@
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
+
+.collapsible {
+  color: $color-gray-dark;
+  cursor: pointer;
+  width: 303px;
+  height: 42px;
+  border: none;
+  text-align: left;
+  outline: none;
+  font-size: 16px;
+  padding: 0;
+  margin: 0;
+  background-color: transparent;
+  border-radius: 0;
+
+  @media (max-width: $small-screen) {
+    width: 100%;
+  }
+
+}
+
+.collapsible:hover {
+  color: $color-gray-dark;
+  background-color: transparent;
+}
+
+.collapsible:active {
+  color: $color-gray-dark;
+  background-color: transparent;
+}
+
+.status-label {
+  pointer-events: none;
+  text-decoration: underline $color-light-blue dotted;
+  text-underline-position: under;
+}
+
+.content {
+  width: 303px;
```

Comment:
Just wondering if we should be using a fixed number of pixels here vs. letting the grid layout handle the width?